Cargo.toml

```diff
@@ -1,2 +1,3 @@
 [workspace]
-members = [ "ingest", "store", "mobile_rewards", "status", "entropy"]
+members = [ "ingest", "store", "mobile_rewards", "status", "entropy",
+"poc_iot_injector"]
```

Comment:
just iot_injector? poc_ seems redundant

Reply:
well.. there'll also be a "packet injector" which injects state channels ?